README.md

```diff
@@ -3,13 +3,52 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+Asymptotic analysis might be misleading for three reasons 
+One is the ignoring of constant factors that still impact actual performance but are not factored into the algorithm's growth rate
+Two the hardware that the algorithm is running on
+Three Other processes are running besides the algorithm that you are testing the performance of.
```

Comment:
Those will affect the runtime, but asymptotic analysis doesn't tell you anything about the runtime itself, just how it grows.